crates/ruma-client-api/src/doc/string_enum.md

```diff
@@ -1,4 +1,5 @@
 <!-- Keep this comment so the content is always included as a new paragraph -->
 This type can hold an arbitrary string. To build this with a custom value, convert it from a
 string with `::from()` / `.into()`. To check for formats that are not available as a
```

Comment:
I think "format" was only valid for a specific instance of this, where the enum represents some sort of format. We should have generalized this as part of extracting the docs into its own file.

```suggestion
string with `::from()` / `.into()`. To check for values that are not available as a
```